packages/eslint-plugin-actual/lib/rules/no-untranslated-strings.js

```diff
@@ -0,0 +1,89 @@
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Disallow non-translated English strings',
+      recommended: true,
+    },
+    schema: [],
+    messages: {
+      noHardcoded: 'Non-translated English string. Wrap in <Trans>.',
+    },
+  },
+  create(context) {
+    const whitelist = [
+      'Actual',
+      'GoCardless',
+      'SimpleFIN',
+      'Pluggy.ai',
+      'YNAB',
+      'nYNAB',
+      'YNAB4',
+
+      '0',
+      '1',
+      '2',
+      '3',
+      '4',
+      '5',
+      '6',
+      '7',
+      '8',
+      '9',
+    ];
+
+    function isProbablyEnglish(text) {
+      const trimmed = text.trim();
+      if (whitelist.includes(trimmed)) {
+        return false;
+      }
+
+      // very basic - but it'll catch most cases
+      return /^[A-Z][a-z].*[a-z]$/.test(trimmed);
```

Comment:
What about strings ending in punctuation? Maybe we should allow optional punctuation marks at the end

Reply:
That'd make sense to include too, added in https://github.com/actualbudget/actual/pull/5212/commits/399e7780f4ebf3728f0f65c498bedb67df1c83b8